### DISCUSSION TITLE
Compatibility with SingleTickerProviderStateMixin

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#76] [Q&A]
I think `RearchConsumer` doesn't implement `SingleTickerProviderStateMixin`. Do you know any workaround to use it?

**Comment by GregoryConrad** [comment]
If you are looking to make an animation controller, see https://pub.dev/documentation/flutter_rearch/latest/flutter_rearch/BuiltinWidgetSideEffects/animationController.html

If you need a single ticker provider directly, see https://pub.dev/documentation/flutter_rearch/latest/flutter_rearch/BuiltinWidgetSideEffects/singleTickerProvider.html

Both can be accessed via `use.fooBar` on a `WidgetHandle`.
